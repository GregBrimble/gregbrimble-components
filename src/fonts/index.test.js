import React from "react"
import Fonts from "./index"
import renderer from "react-test-renderer"

describe("Fonts", () => {
  it("renders properly", () => {
    const tree = renderer.create(<Fonts />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
