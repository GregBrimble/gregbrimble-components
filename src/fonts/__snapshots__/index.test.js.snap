// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fonts renders properly 1`] = `null`;
